package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@hyperjumptech/monika",
   "description": "Synthetic monitoring made easy",
-  "version": "1.17.0",
+  "version": "1.18.0",
```

Comment:
just a toggle is not just an update to patch number (ex:1.17.1)? 😁

Reply:
According to the [Semantic versioning document](https://semver.org/), the minor version needs to be incremented if there is a new backward-compatible functionality. My thinking is, it is still a new functionality even though it is doing nothing.